content/blog/api-console-for-data-services-cloud-console.md

```diff
@@ -0,0 +1,91 @@
+
+Additionally, user can also change the Client Secret to update the authorization when authorized user lost the Client Secret or the secret is compromised. And lastly, when access to the API must be disabled, user can delete the application association in the API Gateway.
+
+![Access API process](/img/user-guide-for-authorization.png "Process to authenticate and to obtain secure access ")
+
+## Accessing the API Gateway Console
+
+To access the API console, user must login into the [GreenLake Common Cloud](https:\common.cloud.hpe.com), deployed the Data Services Common Cloud Console Application, and onboarded a Storage Array (Alletra, Nimble and Primera) into the organization that is associated with the user account. The user must have the role that is required to perform the intended operation at the instance of DSCC where the storage is deployed. e.g. User must have volume management capability in the Data Ops Management to create a storage volume in US West. 
+
+The **API** **Gateway** menu is available inside **Manage** menu. From **Cloud Console** click on **Menu** to access this **Manage** Menu.
+
+![Access Menu ](/img/accesing-manage-menu-from-console.png "Menu in Cloud Console")
+
+![CCS Menu](/img/accesing-api-gateway-from-manage-menu.png "GreenLake Common Cloud Menu")
+
+The API Gateway provides the following operations:
+
+1. Create and manage API client applications association to obtain:
+
+   1. Instance ID of the DSCC at the particular region (Hexadecimals)
+   2. Client ID (Hexadecimals)
+   3. Client Secret (Hexadecimals)
+   4. URL to the HPE GreenLake Cloud Console end-point (string)
+   5. URL to the HPE DSCC end-point (string)
+2. Generate Access Token, change Client Secret, and delete Access Token.
+
+![API Gateway](/img/api-gateway-block.png "DSCC API Gateway")
+
+### Manage API Client Applications
+
+Each of the API client Applications Connection creates the relationship between the client and server (application). Click on the **Create Credentials** button to generate a client credential to generate the Access Token.
+
+![](/img/create-credentials-button.png)
+
+Inside the Create Credentials menu, click on the V button to show the pull down list and use the mouse to click on the desired application. i.e. For storage select the Data Services Cloud Console. The list also show region where the applications is deployed.
+
+![select the application](/img/select-the-desired-application.png "Choose application (DSCC)")
+
+After selecting the correct application enter the Credential Name (Please see HPE GreenLake Cloud Console user guide for supported characters.) Click the Create Credentials button to proceed with Client Credentials creation.
```

Comment:
Add a comma after "After selecting the correct application"
Would it be possible to hotlink the "HPE GreenLake Cloud Console User Guide"? (also, "User Guide" should be initial capped)